control-plane/agents/src/bin/core/node/service.rs

```diff
@@ -119,6 +119,32 @@ impl NodeOperations for Service {
         let node = self.drain(id, label).await?;
         Ok(node)
     }
+
+    /// Apply the label to node.
+    async fn label(
+        &self,
+        id: NodeId,
+        label: Option<HashMap<String, String>>,
```

Comment:
why is label optional? seems None is an error, so we can just make it non-optional